src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -121,6 +121,24 @@ public boolean clickEditInstructorLink(int instrNum) {
         return isEditable;
     }
     
+    public void clickCancelEditInstructorLink(int instrNum) {
+        getCancelEditInstructorLink(instrNum).click();
+    }
+    
+    public boolean isInstructorFormNotEditable(int instrNum) {
```

Comment:
`isInstructorFormEditable` would result in the boolean condition being
```
boolean isEditable = editInstructorNameTextBox.isEnabled()
                     && editInstructorEmailTextBox.isEnabled()
                     && saveButton.isDisplayed();
```
Which means that as long as one of the above elements is not enabled/displayed the form is considered as uneditable, which would also be a false positive.

Perhaps I could rename the method to `isInstructorFormUneditable` or `isInstructorEditFormDisabled` to avoid including `not` in the method name.